### ISSUE TITLE
About Labels of Million-AID Dataset

### ISSUE THREAD

**Issue by redsnappper** [ViTAE-Transformer/ViTAE-Transformer-Remote-Sensing#10]
>> The original split of MillionAID is used for recognition. Our study is about pretraining, so we resplit the training and testing sets. The obtained training set is relatively large for transferring the pretrained weights to downstream tasks. All RSP pretrained weights are available at https://github.com/ViTAE-Transformer/ViTAE-Transformer-Remote-Sensing/blob/main/README.md

>Thank you very much. Your work is very inspiring to us. We are working to do some research in the field of remote sensing image pre-training and would like to use your work as a baseline. However, we found that the open source MillionAID data has less annotated data than in your paper, so we propose this issue.

_Originally posted by @pUmpKin-Co in https://github.com/ViTAE-Transformer/ViTAE-Transformer-Remote-Sensing/issues/3#issuecomment-1123531593_

Hello,
I wasn't able to understand what the conclusion is.

> >  we resplit the training and testing sets.

I think the annotated data is needed to use all images and split the images again. Is the labeled data provided for test data? I mean, in my understanding, we need both "train_label.txt" and "valid_label.txt" to use million-AID, but I don't know where I can download them. I appreciate your help.

**Comment by DotWang** [comment]
Please contact the authors of MillionAID and see the official site for details.

**Comment by DotWang** [comment]
Please contact the authors of MillionAID and see the official site for details.

**Comment by redsnappper** [comment]
I see. Thank you for your response.

**Comment by yyq0828** [comment]
@thanyu-hub Hello, have you got "train_label.txt" and "valid_label.txt"?
